### ISSUE TITLE
MythBox error - Exception: 'NoneType' object has no attribute

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#141]
```
What steps will reproduce the problem?
1. Running the mythbox add-on within XBMC.
2.
3.

What is the expected output? What do you see instead?
It is expected to see the mythbox software run, which worked following initial 
installation. After rebooting, the error was seen and mythbox failed to run.

Screenshot with error and log files are attached.


What version of the product are you using? On what operating system?
Ubuntu 10.10


Please provide any additional information below.

Many thanks for any support that can be provided on this issue. I have been 
through the list of issues and cannot see one that describes this issue.

```

Original issue reported on code.google.com by `sam.w.ba...@gmail.com` on 29 Dec 2010 at 12:10

Attachments:
- [mythbox.log](https://storage.googleapis.com/google-code-attachments/mythbox/issue-141/comment-0/mythbox.log)
- [MythBox Error](https://storage.googleapis.com/google-code-attachments/mythbox/issue-141/comment-0/MythBox Error)
- [xbmc.log](https://storage.googleapis.com/google-code-attachments/mythbox/issue-141/comment-0/xbmc.log)

**Comment by GoogleCodeExporter** [comment]
```
Looks like the settings file is corrupt. Can you please attach 
~/.xbmc/userdata/addon_data/script.mythbox/settings.xml.
```

Original comment by `semir.pa...@gmail.com` on 29 Dec 2010 at 6:17

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the reply, see attached.
```

Original comment by `sam.w.ba...@gmail.com` on 29 Dec 2010 at 6:24

Attachments:
- [settings.xml](https://storage.googleapis.com/google-code-attachments/mythbox/issue-141/comment-2/settings.xml)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 24e02f257ea0.  Thanks for the bug report. As a temporary workaround, 
delete your settings.xml and don't remove 'mythboxfeed' from the list of 
twitter names.

```

Original comment by `semir.pa...@gmail.com` on 29 Dec 2010 at 9:48
- Changed state: **Fixed**
